tools/gulp/constants.ts

```diff
@@ -29,6 +29,12 @@ export const HTML_MINIFIER_OPTIONS = {
   removeAttributeQuotes: false
 };
 
+/** Packages that will be published to NPM by the release task. */
+export const RELEASE_PACKAGES = [
+  'cdk',
+  'material',
+];
```

Comment:
Yeah you would just configure the output folder and the different "tasks" will append the package name for example

```ts
join(DIST_ROOT, 'packages', 'material')
```